### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by kriegalex**
1. Help user notice that tls-san is the same value as entry in /etc/hosts (not sure if it could break k3s service).
2. Help user notice that the overcommit flag is commented because it expects it from another sysctl config.
3. Remind user to check for any modified value before attempt to deploy Nextcloud (for example the domain name).

### CODE REVIEW

File: G911 - Appendix 11 ~ Alternative Nextcloud web server setups.md [hunk 1]
```diff
@@ -668,6 +668,10 @@ The Nexcloud server setup you'll see here will have its own IP through the Metal
     - In the `http.server` subsection, notice the paths specified in the `ssl_certificate` and `ssl_certificate_key` parameters. They'll be used later to mount there the corresponding files from the `wildcard.deimos.cloud-tls` secret you enabled earlier in the `nextcloud` namespace.
 
     - Again in the `http.server` subsection, look for the `root` parameter set there. It has the path, within the Nginx container, to the webroot of your Nextcloud server (`/var/www/html`), where Nextcloud will install all the files it needs to run.
+  
+    > **BEWARE!**  
+    > - It is always a good idea to compare your nginx.conf against the one found [in the Nextcloud documentation](https://docs.nextcloud.com/server/stable/admin_manual/installation/nginx.html), for your version of Nextcloud. Any addition there might be needed in your config too.
```
Comment (ehlesp) [inline G911 - Appendix 11 ~ Alternative Nextcloud web server setups.md:673]: Remove the "-", there's no need for it here in this case (check my review about your second bulleted line). Also, write \`nginx.conf\` (notice the punctuation marks), to give the filename the shell highlighting.

File: G911 - Appendix 11 ~ Alternative Nextcloud web server setups.md [hunk 2]
```diff
@@ -668,6 +668,10 @@ The Nexcloud server setup you'll see here will have its own IP through the Metal
...
 
     - Again in the `http.server` subsection, look for the `root` parameter set there. It has the path, within the Nginx container, to the webroot of your Nextcloud server (`/var/www/html`), where Nextcloud will install all the files it needs to run.
+  
+    > **BEWARE!**  
+    > - It is always a good idea to compare your nginx.conf against the one found [in the Nextcloud documentation](https://docs.nextcloud.com/server/stable/admin_manual/installation/nginx.html), for your version of Nextcloud. Any addition there might be needed in your config too.
+    > - If you get a message like `Error occurred while checking server setup` in the overview screen of the administration section of Nextcloud, carefully look into the logs of your nextcloud pod ([chapter on logs here](https://github.com/kriegalex/smallab-k8s-pve-guide/blob/feature/minor-improvements/G036%20-%20Host%20and%20K3s%20cluster%20~%20Monitoring%20and%20diagnosis.md#checking-the-logs)). For example, you might get an alert log telling you that 8 worker connections is not enough.
```
Comment (ehlesp) [inline G911 - Appendix 11 ~ Alternative Nextcloud web server setups.md:674]: To separate this sentence from the previous one, you just have to add a blank `>` line, as follows:

~~~markdown
>
> If you get a message like...
~~~

This saves you from using bullet points when you want to separate lines in a quote. So please remove the bullet point here too.

**Review by ehlesp — CHANGES_REQUESTED** [review]
Sorry I didn't review your changes sooner.

**Comment by kriegalex** [comment]
I'll review and update this at some point, no worries. Sorry for the many English syntax errors in the doc, it is not my main language. Thanks for the tips.

Context: I've taken a break from k3s since I'm still waiting to build my "real" homelab server that will host k3s. I quickly followed your guide on a very basic system to validate the whole concept.

**Comment by ehlesp** [comment]
> I'll review and update this at some point, no worries. Sorry for the many English syntax errors in the doc, it is not my main language. Thanks for the tips.
> 
> Context: I've taken a break from k3s since I'm still waiting to build my "real" homelab server that will host k3s. I quickly followed your guide on a very basic system to validate the whole concept.

Understood @kriegalex . Don't worry about your english, many in IT are not english speakers (disclaimer, I'm spanish so we are in the same boat). I'll just correct the texts where necessary, but be careful that what you put has been tested and works. Like you, I don't have a K3s system with me right now, although maybe when I have the time I'll create a new one, although with a different setup.

I'll assign you to this pull request since I think it'll be more convenient for future related notifications, hope you don't mind. Don't worry, I won't bother you at all about this. We're not in a rush at all.

Thanks for your contribution!

**Comment by kriegalex** [comment]
I'm closing this pull request as I ended up not following this guide for my final cluster, so I didn't work on it that much. I went with bare metal, K8S and helm instead of Proxmox, K3S and Kustomize.

The only real valuable things from this pull request are:

- nginx-config vs fpm-config in G911
- server vs apache in G033

Thank you for this very good guide, the details help understand things as a beginner.

**Pull request closed without merging**